client/my-sites/stats/activity-log-item/index.jsx

```diff
@@ -79,9 +180,29 @@ class ActivityLogItem extends Component {
 		const {
 			hideRestore,
 			activity: { activityIsRewindable, activityName, activityMeta },
+			plugin,
+			pluginStatus,
+			translate,
 		} = this.props;
 
 		switch ( activityName ) {
+			case 'plugin__update_available':
+				// If there's an update in progress or one was just completed, don't display the button.
+				if ( 'inProgress' === pluginStatus.status || 'completed' === pluginStatus.status ) {
+					return null;
+				}
+				return (
+					plugin.update && (
+						<Button
+							primary
+							compact
+							className="activity-log-item__action"
+							onClick={ this.updateSinglePlugin }
+						>
+							{ translate( 'Update plugin' ) }
```

Comment:
Hi! I've found a possible matching string that has already been translated **88** times:
`translate( 'Update Plugin' )` *ES Score: 16*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).